spec/models/mixins/external_url_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe ExternalUrlMixin do
+  let(:test_class) do
+    Class.new(ActiveRecord::Base) do
+      def self.name; 'TestClass'; end
+      self.table_name = 'vms'
+      include ExternalUrlMixin
+    end
+  end
+
+  describe '#external_url=' do
+    before do
+      User.current_user = FactoryBot.create(:user)
+    end
+
+    let(:test_instance) do
+      i = test_class.create
+      i.external_url = 'https://www.hmpf.cz'
+      i
+    end
+
+    it 'sets url for the current user' do
+      expect(test_instance.external_urls.first.attributes).to include(
+        'resource_type' => 'TestClass',
+        'resource_id'   => test_instance.id,
+        'url'           => 'https://www.hmpf.cz',
+        'user_id'       => User.current_user.id
+      )
+    end
+
+    it 'removes previously set url for the current user' do
+      test_instance.external_url = 'https://www.seznam.cz'
+      test_instance.reload
+      expect(test_instance.external_urls.count).to eq(1)
+      expect(test_instance.external_urls.first.attributes).to include(
+        'resource_type' => 'TestClass',
+        'resource_id'   => test_instance.id,
+        'url'           => 'https://www.seznam.cz',
```

Comment:
Same here, anything in the `example.com` domain is fair game, so `www.other.example.com` would work nicely.

Reply:
ok